nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/layers/convolution/Pooling3DDerivative.java

```diff
@@ -41,9 +41,6 @@ public Pooling3DDerivative(SameDiff sameDiff, SDVariable[] inputs, INDArray[] in
         super(sameDiff, inputs, inputArrays, outputs, inPlace, pooling3DConfig, type);
     }
 
-    public Pooling3DDerivative() {}
```

Comment:
Don't delete the no-arg constructor. Op no-arg constructors are used in tensorflow import - if you ran the full test suite you'd very likely see a test failure as a result of this removing this.